bin/check-java-permgen.rb

```diff
@@ -36,26 +36,27 @@ class CheckJavaPermGen < Sensu::Plugin::Check::CLI
 
   option :warn, short: '-w WARNLEVEL', default: '85'
   option :crit, short: '-c CRITLEVEL', default: '95'
+  option :java, short: '-j <java bin path, including trailing slash>', default: ''
```

Comment:
I think this should be called `java_bin_dir` as when I first saw this I assumed it was the path the java binary. Lets, Also add a `description: java bin path, including trailing slash`.